en/features_mc/throw_launch.md

```diff
@@ -0,0 +1,87 @@
+# Throw Launch (Multicopter)
+
+:::warning Experimental
+This feature was introduced after PX4 v1.14.
+
+- It has not yet been broadly tested on different vehicle configurations or scenarios. various platforms and in different scenarios.
+- The majority of testing has been done in position mode.
+  Other modes should also work.
+
+:::
+
+This feature allows a multicopter to be started by arming it from a fixed position and then throwing it into the air.
+The vehicle then turns on the motors and operates according to its current mode.
+
+The vehicle detects that it has been thrown based on reaching a certain speed and altitude (5m/s and 2m by default, respectively) and then starts the motors at the apex of the throw, once it determines that it has started to fall.
+Note that unlike in other configurations, when throw launch is enabled the propellers should not spin until the launch is detected ( even when the vehicle is armed).
+
+The vehicle will not arm while it is moving, and should not be used on a moving platform (this is to reduce the chance of accidental triggering if a vehicle is armed and then moved to a launch point).
```

Comment:
The first part of the sentence is probably true (I did not modify the arming conditions in any way), but in the initial comment about "moving platform" I was trying to highlight something else. I propose to change this sentence in the following way, please let me know if it's more clear for you now:

> The vehicle should not be transported after being armed and before the throw. In particular, the throw should not be executed from a moving platform. The reason behind this is that the condition to start the motors depends on absolute speed of the multicopter and does not account for any additional movement. Trying to throw the drone from a moving platform might result in the motors being started prematurely.